### PR TITLE
add uuid instead id

### LINKED ISSUE #225: Use uuid as unique IDs to identify data in @id.
### I'm submitting a 
- [ ] bug report.
- [x] feature request.

### Current Behaviour:

Use python Uuid to generate unique IDs instead of the ones assigned by the database to identify data in @id.

@sandeepsajan0  please ask any doubts here. Also, you should use the `agent-uuid` branch from now on.

### Do you want to work on this issue?

No

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Fixes #225 

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [ ] Everything works and tested for Python 3.5.2 and above.

### Description
Update id with uuid for uniqueness and now data type is used by id is the string.
datamodel

### Test Logs
testcrud

#### Added

- Feature 1
Add uuid for unique id.
- Feature 2

#### Changed

- Change 1
- Change 2

#### Fixed

- Bug 1

#### Removed

- Deprecated feature 1

### CODE REVIEW

**Comment by xadahiya** [comment]
Sandeep include description and test logs in PR.

**Comment by sandeepsajan0** [comment]
@xadahiya  done!

**Comment by xadahiya** [comment]
Try adding and fetching some data from the server and post screenshots here, just to make sure it's working

**Comment by sandeepsajan0** [comment]
@xadahiya  I think now it is working fine.
Because the added data at url `/DsCollection/` is:
dataadd
and the data get on url `/DsCollection/uuid`
dataget

File: hydrus/app.py [hunk 1]
```diff
@@ -329,6 +329,7 @@ def get(self, path: str) -> Response:
         """
         Retrieve a collection of items from the database.
         """
+        print("get",path)
```
Comment (xadahiya) [inline hydrus/app.py]: Remove print statements.

File: hydrus/data/db_models.py [hunk 2]
```diff
@@ -18,7 +19,8 @@ class RDFClass(Base):
 
     __tablename__ = "classes"
 
-    id = Column(Integer, primary_key=True)
+    id = Column(String,  default=lambda: str(uuid.uuid4()), unique=True,primary_key=True)
```
Comment (xadahiya) [inline hydrus/data/db_models.py]: PEP format this file properly.

File: hydrus/data/db_models.py [hunk 3]
```diff
@@ -96,7 +100,8 @@ class Terminal(Base):
 
     __tablename__ = "terminals"
 
-    id = Column(Integer, primary_key=True)
+    id = Column(String,  default=lambda: str(uuid.uuid4()), unique=True,primary_key=True)
+#    id = Column(Integer, primary_key=True)
```
Comment (xadahiya) [inline hydrus/data/db_models.py]: Remove old code lines.

**Comment by sandeepsajan0** [comment]
@xadahiya  done!

**Pull request merged**